aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/accessors/dependsOnJsonSetAccessor.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Super cosmetic
but not sure that we need to consider it as something special, it was planed as mostly basic accessor
Mb we need to consider it a default behaviour for object input.

Reply:
moved to the base 'set' accessor